src/wavelet1D.cpp

```diff
@@ -867,7 +867,7 @@ Wavelet1D::calculateSNRatioAndLocalWavelet(const Simbox
       }
       else {
         LogKit::LogFormatted(LogKit::Low, "\n  Not using vertical well %s for error estimation (length=%.1fms  required length=%.1fms).",
-                             blocked_log->GetWellName().c_str(), length*dz_, waveletLength_);
+                             blocked_log->GetWellName().c_str(), length*dz_, waveletLength_+1);
       }
```

Comment:
To avoid message "(length=204 ms, required length 204 ms)". Above we require length*dz_ to be > than waveletLength_.